frontend/public/components/monitoring/alert-manager-yaml-editor.tsx

```diff
@@ -97,9 +98,14 @@ export const AlertManagerYAMLEditorWrapper: React.FC<
   AlertManagerYAMLEditorWrapperProps
 > = React.memo(({ obj, ...props }) => {
   return (
-    <StatusBox {...obj}>
-      <AlertManagerYAMLEditor {...props} obj={obj.data} />
-    </StatusBox>
+    <>
+      <Helmet>
+        <title>Alertmanager YAML</title>
```

Comment:
```suggestion
        <title>Alerting</title>
```

Reply:
Should this also be the case for https://github.com/openshift/console/pull/3255/files?

Reply:
Thanks @spadgett, @dtaylor113. I think #3255 should also use the static `Alerting` to match the page heading, so I've updated that PR.

Reply:
Thanks @spadgett & @kyoto, updated this PR to 'Alerting' as well